### ISSUE TITLE
Login Details

### ISSUE THREAD

**Issue by joelaran13** [NidulaGunawardana/NShop#1]
May i know what the login details?

**Comment by NidulaGunawardana** [comment]
You have to import the following SQL database to your installed computer. Sorry I didn't include it in the repository. In the database you can add any credentials as login in the user table or use existing credentials to log in to the system. When selecting user type in the user table Admin A (1) is for the Admin panel login And Cashier (4) for cashier login. I hope this resolves your issue.

[Nshop_db.zip](https://github.com/user-attachments/files/15712191/Nshop_db.zip)

**Comment by joelaran13** [comment]
thanks for the guide, really appreciate it.

**Comment by joelaran13** [comment]
What the name of the database? Is it "nshop" or "Nshop_db" ?

**Comment by NidulaGunawardana** [comment]
Yes it is "nshop",  Nshop_db is just the exported file name. You just need to execute the sql file it will create the database as well.

**Comment by joelaran13** [comment]
i see, i've done that but i can't log in eventho i know the username and
the password.

**Comment by joelaran13** [comment]
java.sql.SQLException: Access denied for user 'root'@'localhost' (using
password: YES)

it says this, why is it denied me?

**Comment by NidulaGunawardana** [comment]
This error comes because of the database login details are wrong in your system. you have to enter your database password in the following directory in your project.
![Screenshot 2024-06-07 140130](https://github.com/NidulaGunawardana/NShop/assets/94860959/a47087e6-3761-495e-88ea-1cf154e043d1)
You have to update the password to your database password in the src/model/MySQL.java File

**Comment by joelaran13** [comment]
[image: localhost _ 127.0.0.1 _ phpMyAdmin 5.2.1 - Google Chrome 07_06_2024
16_42_31.png]

**Comment by joelaran13** [comment]
you mean here on the database?

**Comment by joelaran13** [comment]
it okay i got it

**Comment by NidulaGunawardana** [comment]
Glad to hear that. if any issue let me know.

**Comment by joelaran13** [comment]
Hello, good days. i encounter another problems here... Which is the
dashboard doesnt appear total sale and etc. it just blank. Got any idea on
how to fix it?

**Comment by joelaran13** [comment]
Hello, my dashboard doesn't appear the total sales, etc. It just blank.

**Comment by NidulaGunawardana** [comment]
After you do sales it will appear automatically in there, You can sell products in cashiers section then they will appear.

**Comment by joelaran13** [comment]
Hello, good day to you. I still have the same problem with the dashboard,
it still blank, total blank like there nothing there.

**Comment by joelaran13** [comment]
[image: Test.png]

This is the dashboard in the software. There nothing here i believe.

**Comment by joelaran13** [comment]
can you send me the "dashboard.java" file, so i can compared it.
